src/mir/dataflow.md

```diff
@@ -89,7 +89,21 @@ statement must be considered separately from the left-hand side.
 
 ### Convergence
 
-TODO
+Your analysis must converge to "fixpoint", otherwise it will run forever.
+Converging to fixpoint is just another way of saying "reaching equilibrium".
+In order to reach equilibrium, your analysis must obey some laws. One of the
+laws it must obey is that the bottom value joined with some other value equals
+the second value. Or, as an equation:
```

Comment:
Why is a bottom value necessary? I see that having a top is important, since the top is a fixpoint, but what is the bottom used for?

Reply:
What would be the initial state of the analysis if you didn't have a bottom value? The bottom value *is* that initial state.

Reply:
Got it, thanks - I didn't realize 'bottom' meant 'initial state'. Could you add that?

Reply:
Sure, good idea. Although I think the bottom value is a bit more general than 'initial state'. Not 100% sure though.